### PR TITLE
fc-0068 docs: documentation style guide

### PR DESCRIPTION
**Pull request by jesgreco**
Adding the new Open edX Documentation Writing Style Guide

Google Drive Doc: https://docs.google.com/document/d/13QbAoT1N4rac41r90YJImPzMTuY4rpqH5EnyPO8sw4w/edit?tab=t.0#heading=h.qm0lcv76r67g

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @jesgreco!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.



#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?


This repository is currently maintained by `@openedx/docs-openedx-org-maintainers`.  Tag them in a comment and let them know that your changes are ready for review.


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

File: source/documentors/references/doc_style_guide.rst
Comment (Apgomeznext) [inline source/documentors/references/doc_style_guide.rst:192]: .. code-block::

  .. image:: _assets/example_image_2.png
      :width: 500px
      :align: center
      :alt: Alternative text. this is important for accessibility

File: source/documentors/references/doc_style_guide.rst
Comment (sarina) [reply source/documentors/references/doc_style_guide.rst:320]: Which example?

File: source/documentors/references/doc_style_guide.rst
Comment (sarina) [inline source/documentors/references/doc_style_guide.rst:354]: I think you're missing an intro to the bulleted list, maybe, "When to use a table"?

File: source/documentors/references/doc_style_guide.rst
Comment (sarina) [inline source/documentors/references/doc_style_guide.rst:359]:
```suggestion
When not to use a table:

```

Check rendering please!

File: source/documentors/references/doc_style_guide.rst
Comment (sarina) [inline source/documentors/references/doc_style_guide.rst:363]: `:ref:` link to the RST guide here.

File: source/documentors/references/doc_style_guide.rst
Comment (sarina) [inline source/documentors/references/doc_style_guide.rst:372]:
```suggestion
The syntax of tags is a directive, `.. tags::`, followed by the tag names, such as `educators, reference`. It renders as follows:
```

File: source/documentors/references/doc_style_guide.rst
Comment (sarina) [inline source/documentors/references/doc_style_guide.rst:383]:
```suggestion
The Open edX ``docs.openedx.org`` repository is using a documentation markup language called reStructuredText (RST) which is very similar to Markdown. The following guide is helpful to refer to as you produce RST documentation: RST Guide.
```

Comment (sarina) [reply source/documentors/references/doc_style_guide.rst:383]: `:ref:` link to the RST guide

File: source/documentors/references/doc_style_guide.rst
Comment (sarina) [inline source/documentors/references/doc_style_guide.rst:388]: This link is wrong, we are getting rid of this site. Should this refer to another guide in docs.openedx.org?

File: source/documentors/references/doc_style_guide.rst
Comment (sarina) [inline source/documentors/references/doc_style_guide.rst:399]:
```suggestion
• Use the `` :guilabel:`` directive to highlight the text of a button.
```

File: source/documentors/references/doc_style_guide.rst
Comment (sarina) [inline source/documentors/references/doc_style_guide.rst:406]:
```suggestion
*Italic* text can be used to add emphasis to a word, phrase or sentence. As italic text is slightly more difficult to read, it should be used sparingly. Instead, we recommend using bolded text. 
```

Suggest putting the word in its formatting

File: source/documentors/references/doc_style_guide.rst
Comment (sarina) [inline source/documentors/references/doc_style_guide.rst:411]:
```suggestion
Using **bold type** for emphasis serves to draw the reader's attention to specific words or phrases that are particularly important or impactful.
```
Suggest putting the word in its formatting

**Review by sarina — CHANGES_REQUESTED** [review]

File: source/documentors/references/doc_style_guide.rst
Comment (jesgreco) [reply source/documentors/references/doc_style_guide.rst:389]: Yes, I've just replaced it.

File: source/documentors/references/doc_style_guide.rst
Comment (jesgreco) [reply source/documentors/references/doc_style_guide.rst]: Done! I've included all RST guide's links.

File: source/documentors/references/doc_style_guide.rst
Comment (jesgreco) [reply source/documentors/references/doc_style_guide.rst:322]: I'm deleting such sentence.

File: source/documentors/references/doc_style_guide.rst
Comment (jesgreco) [reply source/documentors/references/doc_style_guide.rst:354]: Yes! I've added!

**Comment by jesgreco** [comment]
I've made all the fixes requested by @sarina
Let me know if there is anything else to be fixed.

File: source/documentors/references/doc_style_guide.rst
Comment (sarina) [inline source/documentors/references/doc_style_guide.rst:372]:
```suggestion
The syntax of tags is a directive, `.. tags::`, followed by the tag names, such as `educators, reference`. It renders as follows:
```
```suggestion
The syntax of tags is a directive, ``.. tags::``, followed by the tag names, such as `educators, reference`. It renders as follows:
```

This will fix one of your build errors, can you check out the other reasons why your build is failing?

**Review by jesgreco — COMMENTED** [review]
@sarina @Apgomeznext @jswope00 Can you please help me with the error? I've just edited the doc and still the error is there. Thanks!

File: source/documentors/references/doc_style_guide.rst
Comment (sarina) [inline source/documentors/references/doc_style_guide.rst:100]: Can you link this to the appropriate document or section?

File: source/documentors/references/doc_style_guide.rst
Comment (sarina) [inline source/documentors/references/doc_style_guide.rst:105]: Maybe link to the document
```suggestion
Open edX documentation is written in American English. See :doc:`doc_english_writing`.
```

File: source/documentors/references/doc_style_guide.rst
Comment (sarina) [reply source/documentors/references/doc_style_guide.rst:383]: This is still the wrong link, we should not link to the draft-edx-style-guide

File: source/documentors/references/doc_style_guide.rst
Comment (sarina) [inline source/documentors/references/doc_style_guide.rst:394]:
```suggestion
• Use the ``:guilabel:`` directive to highlight the text of a button.
```

Comment (sarina) [reply source/documentors/references/doc_style_guide.rst:394]: @jesgreco can you address this comment?

File: source/documentors/references/doc_style_guide.rst
Comment (sarina) [reply source/documentors/references/doc_style_guide.rst:105]: @jesgreco can you address this comment?

File: source/documentors/references/doc_style_guide.rst
Comment (sarina) [reply source/documentors/references/doc_style_guide.rst:100]: @jesgreco can you address this comment?

File: source/documentors/references/doc_style_guide.rst
Comment (sarina) [reply source/documentors/references/doc_style_guide.rst:383]: @jesgreco can you address this comment?

File: source/documentors/references/doc_style_guide.rst
Comment (jesgreco) [reply source/documentors/references/doc_style_guide.rst]: Done!

File: source/documentors/references/doc_style_guide.rst
Comment (jesgreco) [reply source/documentors/references/doc_style_guide.rst]: Done!

File: source/documentors/references/doc_style_guide.rst
Comment (jesgreco) [reply source/documentors/references/doc_style_guide.rst]: Done :)

**Comment by jesgreco** [comment]
@sarina all comments were addressed :) I've just finished reviewing the doc, and adjusted notes and links. Let me know.

**Pull request merged**
